### ISSUE TITLE
peristaltic pump code for experimental design

### ISSUE THREAD

**Issue by monroews** [AguaClara/aguaclara_research#5]
We need code that walks students through the process of selecting peristaltic pump tubing, stock concentrations, and flow rates. The ideas for this are in the [auto tutorial for peristaltic pumps](https://confluence.cornell.edu/display/AGUACLARA/Auto+Tutorial+for+Peristaltic+Pumps)  and a [Mathcad sheet has code](http://designserver.cee.cornell.edu/aguaclaracode/Stock%20chemical%20design%20for%206%20roller%20pumps.xmcd) that can be a guide. 

The code should include a function that returns uL/rev for the different sizes of peristaltic tubing.
Perhaps that is most critical function at this time. The additional analysis can be recreated by the teams as they design their apparatus.

**Comment by fletchapin** [comment]
@monroews I just finished converting the Mathcad file and pushed to the tube_sizing branch.

The one question I was unsure of is when calculating stock concentrations we discussed using only the maximum values, however in Mathcad you had calculated the maximum stock concentration using the minimum concentration along with minimum RPM as shown below.

![pacl_stock_calc](https://user-images.githubusercontent.com/17417538/40629133-6919cf38-6297-11e8-89ff-63348a7d45c2.PNG)

Should I use the maximum or minimum concentrations when calculating C_PACL_stock_max with the minimum RPM?
